changelog/17160.txt

```diff
@@ -1,3 +1,3 @@
 ```release-note:improvement
-auth/gcp: bump github.com/hashicorp/vault-plugin-auth-gcp to v0.14.0
+auth/gcp: Updates dependencies: `github.com/hashicorp/vault-plugin-auth-gcp`.
```

Comment:
I think Austin meant for the change to actually list the improvements and bug fixes from the plugin, as they are improvements here too. At least that's how I interpreted a similar situation with my PR to update kerberos auth.

Example:
```
auth/gcp: Enables GCP roles to be compatible with Vault's role based quotas, add support for GCE regional instance groups, updates dependencies google.golang.org/api to v0.83.0 and github.com/hashicorp/go-gcp-common to v0.8.0
```

@austingebauer Please correct me if I'm misleading here.

Reply:
Yep, that's correct @robmonte. I think in this case, the first two in [this list](https://github.com/hashicorp/vault-plugin-auth-gcp/blob/main/CHANGELOG.md#v0140) already have changelog entries in Vault. We just need one for the dep updates (exactly similar to https://github.com/hashicorp/vault/pull/17174/files#diff-7c432b8c3bb8887f5e0b389cbb81f2bdf4a6964f0a42dc395a6ed5aaf639c7fc)

Reply:
Ah, sorry for the confusion! Fixed in latest commit